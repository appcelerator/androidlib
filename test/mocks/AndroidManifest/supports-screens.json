{
	"supports-screens": {
		"android:resizeable": true,
		"android:smallScreens": true,
		"android:normalScreens": true,
		"android:largeScreens": true,
		"android:xlargeScreens": true,
		"android:anyDensity": false,
		"android:requiresSmallestWidthDp": 320,
		"android:compatibleWidthLimitDp": 480,
		"android:largestWidthLimitDp": 2048
	}
}
