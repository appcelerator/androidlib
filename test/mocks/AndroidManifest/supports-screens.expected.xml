<?xml version="1.0" encoding="utf-8"?>
<manifest>
  <supports-screens android:anyDensity="false" android:compatibleWidthLimitDp="480" android:largeScreens="true" android:largestWidthLimitDp="2048" android:normalScreens="true" android:requiresSmallestWidthDp="320" android:resizeable="true" android:smallScreens="true" android:xlargeScreens="true"/>
</manifest>
